microsoft-edge/webview2/releasenotes.md

```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
+*   Updated the [WebResourceResponseReceived][ReferenceWin32Icorewebview2experimentalAddWebresourceresponsereceived10674] event, and added immutable [WebResourceResponseView][ReferenceWin32Icorewebview2experimentalwebresourceresponseview10674] and [WebResourceResponseReceivedEventArgs::PopulateResponseContent][ReferenceWin32Icorewebview2experimentalwebresourceresponsereceivedeventargsPopulateresponsecontent09628] to become [WebResourceResponseView::GetContent][ReferenceWin32Icorewebview2experimentalwebresourceresponseviewGetcontent10674].
+*   Turned off [Microsoft Defender Application Guard (WDAG)][WindowsSecurityThreatProtectionMicrosoftDefenderApplicationGuardWindows10] in WebView2.
+*   Added [SystemCursorId][ReferenceWin32Icorewebview2experimentalcompositioncontroller2GetSystemcursorid10674] for Visual Hosting.
+*   Input Method Header is now supported in visual hosting.
+*   Developers no longer have to include version.lib when using WebView2 static lib
+ 
+#### .NET  
+
+*   Updated [CoreWebView2][DotnetApiMicrosoftWebWebview2CoreCorewebview2] class to expose CoreWebView2Environment variables.  
+*   Changed implementations of custom EventArgs classes in `Microsoft.Web.WebView2.Core` namespace to be subclasses of [System.EventArgs][DotnetApiSystemEventargs] or [System.ComponentModel.CancelEventArgs][DotnetApiSystemComponentmodelCancelEventargs]. \([\#250][GithubMicrosoftedgeWebviewfeedbackIssue250]\)  
+*   Added support for [CoreWebView2CreationProperties][DotnetApiMicrosoftWebWebview2WinformsCorewebview2creationproperties] in WinForms. \([\#204][GithubMicrosoftedgeWebviewfeedbackIssue204]\)
+*   Added .NET [WebResourceRequested][DotnetApiMicrosoftWebWebview2CoreCorewebview2Webresourcerequested] APIs.  \([\#219][GithubMicrosoftedgeWebviewfeedbackIssue219]\).  
+*   Updated WinForms Designer [Source][DotnetApiMicrosoftWebWebview2WinformsWebview2Source] property to default or reset to null. \([\#177][GithubMicrosoftedgeWebviewfeedbackIssue177]\).  
+*   Updated WebView2 bounds in WebView2.Init() to support DPI modes that are less than 100%.  \([\#432][GithubMicrosoftedgeWebviewfeedbackIssue432]\).  
+*   Updated [BuildWindowCore][DotnetApiMicrosoftWebWebview2WpfWebview2Buildwindowcore] and [DestroyWindowCore][DotnetApiMicrosoftWebWebview2WpfWebview2Destroywindowcore] to be more robust.  \([\#382][GithubMicrosoftedgeWebviewfeedbackIssue382]\).  
+*   Updated .NET Loader base to load on process bit instead of operating system architecture. \([\#431][GithubMicrosoftedgeWebviewfeedbackIssue431]\).  
+ 
+## 1.0.622.22  
+
+Release Date: Oct 19, 2020  
+
+[NuGet package][NuGetGallery1.0.622.22] \| minimum WebView2 Runtime version 86.0.622.22.  
+
+#### General  
+
+> [!IMPORTANT]
+> **Announcement**: This release is now Generally Available (GA).
```

Comment:
> This release is now Generally Available (GA).
Recommend `Win32 C/C++ WebView2 is now Generally Available (GA). Starting with this release, Release SDKs will be forward-compatible. See [GA announcement blog post]() for more details.`

I don't have the link to the post before it goes live, but you can make an aka.ms link here and redirect to the blog once it's live. Let @peiche-jessica  or me know if you don't know how to do that.

Reply:
Something like `aka.ms/wv2gablogpost`? Where should I point it to?

Reply:
https://blogs.windows.com/msedgedev/ until the blog is live